src/components/organisms/NavBarSchedule/NavBarSchedule.tsx

```diff
@@ -134,13 +139,32 @@ export const NavBarSchedule: FC<NavBarScheduleProps> = ({ isVisible }) => {
     };
   }, [relatedVenueEvents, userEventIds, selectedDayIndex, getEventLocation]);
 
+  const downloadCalenderPersonal = useCallback(() => {
+    const dayStart = addDays(startOfToday(), selectedDayIndex);
+    const allPersonalEvents: PersonalizedVenueEvent[] = relatedVenueEvents
+      .map(prepareForSchedule(dayStart, userEventIds, "calender"))
+      .filter((event) => event.isSaved);
+
+    createCalendar(allPersonalEvents, "Personal");
+  }, [relatedVenueEvents, userEventIds, selectedDayIndex]);
+
+  const downloadCalenderAll = useCallback(() => {
+    createCalendar(relatedVenueEvents, "Full");
```

Comment:
recommendation: prefix with `PLATFORM_BRAND_NAME` from `src/settings.ts` (which will be `OHBM2021` in `env/ohbm` branch) and `_`.  This way filename would be more descriptive.  Similarly for personal schedule

Reply:
I was wondering if that's possible! Thanks for pointing out where this info sits!